### PR TITLE
added feature/registrationFront

### PR DESCRIPTION
**Pull request by LukaszSzetela**

### CODE REVIEW

File: src/main/resources/templates/Registration.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    <link rel="stylesheet" type="text/css" href="css/registration.css">
+
+    <!-- Website Font style -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
+
+    <!-- Google Fonts -->
+    <link href='https://fonts.googleapis.com/css?family=Passion+One' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Oxygen' rel='stylesheet' type='text/css'>
+
+    <title>Admin</title>
```
Comment (RafalCiolek) [inline src/main/resources/templates/Registration.html]: Maybe 'Admin new user' insted of Admin? :)

File: src/main/resources/templates/Registration.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+</head>
+<body>
+<div class="container">
```
Comment (RafalCiolek) [inline src/main/resources/templates/Registration.html]: pls add header

File: src/main/resources/templates/Registration.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                </div>
+
+                <div class="form-group ">
+                    <button type="button" class="btn btn-primary btn-lg btn-block login-button">Register</button>
+                </div>
+            </form>
+        </div>
+    </div>
+</div>
+
```
Comment (RafalCiolek) [inline src/main/resources/templates/Registration.html]: pls add footer

File: src/main/resources/templates/Registration.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+<script type="text/javascript" src="assets/js/bootstrap.js"></script>
```
Comment (RafalCiolek) [inline src/main/resources/templates/Registration.html]: I don't see this file. It is needed?

**Review by snafx — APPROVED** [review]

File: src/main/resources/templates/Registration.html [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="assets/css/bootstrap.css">
+
+    <!-- Website CSS style -->
+    <link rel="stylesheet" type="text/css" href="css/registration.css">
+
+    <!-- Website Font style -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
```
Comment (RafalCiolek) [inline src/main/resources/templates/Registration.html]: boostrap version in not consist, check index example. Consider with our group, copy bootstrap file into this project and use it in all views.

File: src/main/resources/templates/registration.html [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
+
+    <!-- Google Fonts -->
+    <link href='https://fonts.googleapis.com/css?family=Passion+One' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Oxygen' rel='stylesheet' type='text/css'>
+
+    <title>Admin</title>
+</head>
+<body>
+<div class="container">
```
Comment (RafalCiolek) [inline src/main/resources/templates/registration.html]: pls add header

File: src/main/resources/templates/registration.html [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+                </div>
+
+                <div class="form-group ">
+                    <button type="button" class="btn btn-primary btn-lg btn-block login-button">Register</button>
+                </div>
+            </form>
+        </div>
+    </div>
+</div>
+
```
Comment (RafalCiolek) [inline src/main/resources/templates/registration.html]: pls add footer

File: src/main/resources/templates/registration.html [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+<script type="text/javascript" src="assets/js/bootstrap.js"></script>
```
Comment (RafalCiolek) [inline src/main/resources/templates/registration.html]: I don't see this file. Is it needed?

File: src/main/resources/templates/registration.html [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="assets/css/bootstrap.css">
+
+    <!-- Website CSS style -->
+    <link rel="stylesheet" type="text/css" href="css/registration.css">
+
+    <!-- Website Font style -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
```
Comment (RafalCiolek) [inline src/main/resources/templates/registration.html]: Inconsistent wersion of boostrap, check index file. Consider (with out group) copy bootstrap file into this project, and use it in all views.

File: src/main/resources/templates/registration.html (hunk 7 above)
Comment (LukaszSzetela) [reply src/main/resources/templates/registration.html]: I added

File: src/main/resources/templates/Registration.html (hunk 5 above)
Comment (LukaszSzetela) [reply src/main/resources/templates/Registration.html]: Done

**Review by RafalCiolek — APPROVED** [review]

**Pull request merged**